<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>CRISPR gene drive system</title>
</head>

<body>
    <h1>Effect of CRISPER gene drive system in the wild population</h1>
    <p>In this article the researcher produced a mathematical model to calculate efficiency of gene drive system is
        effcted by the drive resistant allel. Presence of drive-resistant allele reduces the efficiency of CRISPR-based
        gene drive systems. Mathematical tools were developed to analyse the efficiency of the gene drive system. The
        aim is to quantify the possibility and extent of the spread of the gene drive system.
        One of the crucial steps of the CRISPR-based gene drive system is the breaking of the double strand. The break
        in the double strand is repaired by either of the two way, homology directed repair or non-homologous repair
        pathway. When the nonhomologous pathway is the used for the repairing of the broken DNA, there is a possibility
        of creating mutation due to the mechanism.
        <br><br>
    <figure>
        <img src="pages /crispr2018.jpg" alt="">
        <figcaption>Existing alteration-type CRISPR gene drive systems should invade well-mixed wild populations. (A)
            Typical construction and function of alteration-type CRISPR gene drive systems. A drive construct (D),
            including a
            CRISPR nuclease, guide RNA (gRNA), and ‘cargo’ sequence, induces cutting at a wild-type allele (W) with
            homology to sequences flanking the drive construct. Repair by homologous recombination (HR) results in
            conversion of the wild-type to a drive allele, or repair by nonhomologous end-joining (NHEJ) produces a
            drive-
            resistant allele (R). (B) Drives are predicted to invade by deterministic models when the fitness of DW
            heterozygotes, f , and the homing efficiency, P, are in the shaded region. Vertical lines indicate empirical
            efficiencies from Appendix 1—table 1. (C) Diagram of a single step of the gene-drive Moran process. (D)
            Finite-
            population simulations of 15 drive individuals released into a wild population of size 500, assuming
            conservative
            (P ¼ 0:5) or high (P ¼ 0:9) homing efficiencies, as well as a low-efficiency, constitutively active system
            (P ¼ 0:15).
            Individual sample simulations (solid lines), and 50% confidence intervals (shaded), calculated from 103
            simulations.
            Drive-allele frequencies red and resistant-allele frequencies blue. Peak drive, or maximum frequency
            reached, is
            illustrated by dashed lines and arrows. (E) Peak drive distributions and medians with varying numbers of
            individual
            organisms released (P ¼ 0:5). (F) Medians of peak drive distributions for varying homing efficiencies (P ¼
            0:15,
            bottom; P ¼ 0:5, middle; P ¼ 0:9, top). Throughout, we assume neutral resistance (fWR ¼ fRR ¼ 1) and a 10%
            dominant drive fitness cost (fWD ¼ fDD ¼ fDR ¼ 0:9) (Nobel et al., 2018). </figcaption>

    </figure>

    <br>
    <p>
        content and image reference can be found here: https://elifesciences.org/articles/33423
    </p>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="stylingbio/style.css">
</body>


</html>